### PR TITLE
wdm 0.2.4 :snowflake:

### PR DESCRIPTION
**Pull request by lorepirri**
wdm 0.2.4 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5115]
- dev_url: https://github.com/tnagler/wdm/tree/5337860a9a77f147f3adebf4c93fe2ca74d5a028

### Explanation of changes:

- headers only package
- simple test, headers tests

### Dependencies

- dependency for AnacondaRecipes/pyvinecopulib-feedstock#1

[PKG-5115]: https://anaconda.atlassian.net/browse/PKG-5115?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ViridianMelody — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+
+test:
+  requires:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]: Do you need ninja and cmake?

Comment (lorepirri) [reply recipe/meta.yaml]: Ah, no I don't use them for the test, nice catch, thank you

Comment (lorepirri) [reply recipe/meta.yaml]: I've also removed the build step in the scripts. I left configure+install

**Review by JeanChristopheMorinPerso — APPROVED** [review]
You might want to test using this package in the other package before merging.

**Comment by lorepirri** [comment]
> You might want to test using this package in the other package before merging.

Yes, it seems to work (osx failed here https://github.com/AnacondaRecipes/pyvinecopulib-feedstock/pull/1/commits/ec52532d5b57f83e009c726314496850d03770a1 for a `sigabort`) 

rebuilding now https://github.com/AnacondaRecipes/pyvinecopulib-feedstock/pull/1

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**